client/my-sites/stats/stats-navigation/index.jsx

```diff
@@ -34,7 +34,7 @@ const StatsNavigation = ( props ) => {
 			statsControl = (
 				<SegmentedControl
 					// eslint-disable-next-line wpcalypso/jsx-classname-namespace
-					className="stats-navigation__control"
+					className="stats-navigation__control for-woocommerce"
```

Comment:
> First due to the async load (if I'm not mistaken)

Yeah, good point. Or it will unnecessarily bring in the async file. Will move it to SegmentControl.